src/tr/tr.rs

```diff
@@ -47,24 +50,28 @@ fn delete(set: ExpandSet, complement: bool) {
         }
     };
 
-    let mut reader = BufReader::new(stdin());
+    stdin.lock();
```

Comment:
I expected the lock to be released the moment read_line finishes execution, so that it would have to be re-acquired when reading the next line; this is really unnecessary and will hurt performance. [Source](https://www.reddit.com/r/rust/comments/3rj54u/how_can_i_read_char_by_char_from_stdin/cwpojn1/)